### PR TITLE
Add first custom cops

### PR DESCRIPTION
**Pull request by burisu**
- Install [`rubocop-github`](https://github.com/github/rubocop-github) as default config
- Add `Codeur/RailsAppPatterns` cop
- Add `Codeur/RailsAvoidInstanceMethodsInHelpers` cop
- Adjust gem structure
- Add GitHub Actions workflows: `test.yml` and `quality.yml` (brakeman + rubocop)

### CODE REVIEW

File: default.yml [hunk 1]
```diff
@@ -20,6 +25,16 @@ AllCops:
 Rails:
   Enabled: true
 
+Codeur/RailsAppPatterns:
+  Enabled: true
+  ForbiddenPatterns:
+    - presenters
+    - view_objects
+    - modules
```
Comment (KevinBerthier) [inline default.yml:33]: Do we really want to forbid modules ? Where would you move it instead ?

**Comment by KevinBerthier** [comment]
Also can we disable https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Lint/MissingSuper ?

File: default.yml (hunk 1 above)
Comment (burisu) [reply default.yml:33]: Most of them seems to be created as concerns (before concerns exist in Rails).
To be clear, I talk about `app/modules` in Codeur, and not the modules in general.

**Comment by burisu** [comment]
> Also can we disable https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Lint/MissingSuper ?

Yes, I'll change that

File: default.yml (hunk 1 above)
Comment (KevinBerthier) [reply default.yml:33]: Ok perfect

**Review by KevinBerthier — APPROVED** [review]

**Pull request merged**